ID: PR

DOTzX opened the pull request.

DOTzX committed:
ID: Initial

README.md
common, done.
config, ongoing.

DOTzX committed:
ID: Fix Typo

DOTzX committed:
ID: Few Update config.csv

KiriyaNico committed:
Update and rename Files/accident.csv to Files/Battle Arena/accident.csv

エンブレムフィーバー = Emblem Fever
エンブレムを回収せよ = Kumpulkan Emblem

KiriyaNico committed:
Rename Files/Battle Arena/accident.csv to Battle Arena/accident.csv

KiriyaNico committed:
Rename Battle Arena/accident.csv to Dialouge/accident.csv

KiriyaNico committed:
Update and rename Files/go01000000.csv to Files/Orders/Team/go01000000.csv

KiriyaNico committed:
Rename Files/Orders/Team/go01000000.csv to Orders/Team/go01000000.csv

KiriyaNico committed:
Update go01000000.csv

KiriyaNico committed:
Update go01000001.csv

KiriyaNico committed:
Rename Files/go01000001.csv to Orders/Team/go01000001.csv

KiriyaNico committed:
Update go01000002.csv

KiriyaNico committed:
Rename Files/go01000002.csv to Orders/Team/go01000002.csv

KiriyaNico committed:
Update and rename Files/go01000003.csv to Orders/Team/go01000003.csv'

KiriyaNico committed:
Rename go01000003.csv' to go01000003.csv

KiriyaNico committed:
Update and rename Files/go01000004.csv to Orders/Team/go01000004.csv

KiriyaNico committed:
Update and rename Files/go01000005.csv to Orders/Team/go01000005.csv

KiriyaNico committed:
Update and rename Files/go01000006.csv to Orders/Team/go01000006.csv

KiriyaNico committed:
Update and rename Files/go01000007.csv to Orders/Team/go01000007.csv

KiriyaNico committed:
Update and rename Files/go01000008.csv to Orders/Team/go01000008.csv

KiriyaNico committed:
Update and rename Files/go01000009.csv to Orders/Team/go01000009.csv

KiriyaNico committed:
Update and rename Files/go01001000.csv to Orders/Team/go01001000.csv

KiriyaNico committed:
Update and rename Files/go01001001.csv to Orders/Team/go01001001.csv

KiriyaNico committed:
Update and rename Files/go01001002.csv to Orders/Team/go01001002.csv

KiriyaNico committed:
Update and rename Files/go01001003.csv to Orders/Team/go01001003.csv

KiriyaNico committed:
Update go01001002.csv

KiriyaNico committed:
Update and rename Files/go01001004.csv to Orders/Team/go01001004.csv

KiriyaNico committed:
Update and rename Files/go01001005.csv to Orders/Team/go01001005.csv

KiriyaNico committed:
Update and rename Files/go01001006.csv to Orders/Team/go01001006.csv

KiriyaNico committed:
Update and rename Files/go01001007.csv to Orders/Team/go01001007.csv

KiriyaNico committed:
Update and rename Files/go01001009.csv to Orders/Team/go01001009.csv

KiriyaNico committed:
Update and rename Files/go01001010.csv to Orders/Team/go01001010.csv

KiriyaNico committed:
Update go01001000.csv

KiriyaNico committed:
Update and rename Files/go01002000.csv to Orders/Team/go01002000.csv

KiriyaNico committed:
Update and rename Files/go01002001.csv to Orders/Team/go01002001.csv

KiriyaNico committed:
Update and rename Files/go01002002.csv to Orders/Team/go01002002.csv

KiriyaNico committed:
Update and rename Files/go01002003.csv to Orders/Team/go01002003.csv

KiriyaNico committed:
Update and rename Files/go01002004.csv to Orders/Team/go01002004.csv

KiriyaNico committed:
Update and rename Files/go01002005.csv to Orders/Team/go01002005.csv

KiriyaNico committed:
Update and rename Files/go01002006.csv to Orders/Team/go01002006.csv

KiriyaNico committed:
Update and rename Files/go01002007.csv to Orders/Team/go01002007.csv

KiriyaNico committed:
Update and rename Files/go01002008.csv to Orders/Team/go01002008.csv

KiriyaNico committed:
Update and rename Files/go01002009.csv to Orders/Team/go01002009.csv

DOTzX committed:
Merge pull request #1 from KiriyaNico/ID

ID: Restore.

DOTzX committed:
ID: Finish Remain

DOTzX committed:
ID: Update

DOTzX committed:
ID Update 2

DOTzX committed:
ID: Translate

DOTzX committed:
ID: Translate 2

DOTzX committed:
ID: Reorganizing File

DOTzX committed:
ID: Few Update

DOTzX committed:
ID: Fix unexpected end of data

DOTzX committed:
Update README.md

DOTzX committed:
ID: Fix Again

DOTzX committed:
ID: Whitespace ?!

DOTzX committed:
ID: Aspell pls.

*re-arrange from alama's PR*

DOTzX committed:
ID: Last Fix ?

DOTzX committed:
ID I think

DOTzX committed:
ID: Update README.md

alama committed:
Merge remote-tracking branch 'upstream/ID' into ID

alama committed:
ID: remove wide EN strings

alama commented on UI/common.csv:
```diff
@@ -0,0 +1,1244 @@
+LobbyAction_comic#0,"""漫才"""
+LobbyAction_present#0,"""Mempersembahkan"""
+LobbyAction_water_a#0,"""Menciprat air"""
+LobbyAction_water_b#0,"""Ｔ\u3000ｅ\u3000ｒ\u3000ｃ\u3000ｉ\u3000ｐ\u3000ｒ\u3000ａ\u3000ｔ"""
```
can you remove the fullwidth space (\u3000) and convert the `Terciprat`

alama commented on UI/common.csv:
```diff
@@ -0,0 +1,1244 @@
+LobbyAction_kashige#0,"""Memiringkan kepala"""
+LobbyAction_hello2#0,"""Menyambut 2"""
+LobbyAction_aid2#0,"""Bersorak 2"""
+LobbyAction_pains2#0,"""Ｔ\u3000ｅ\u3000ｒ\u3000ｋ\u3000ｕ\u3000ｔ\u3000ｕ\u3000ｋ"""
```
can you remove the fullwidth space (\u3000) and convert the `Terkutuk`

alama commented on UI/common.csv:
```diff
@@ -0,0 +1,1244 @@
+LobbyAction_line#0,"""Garis"""
+LobbyAction_pose14#0,"""ポーズ１３"""
+LobbyAction_halloween#0,"""Permen dong"""
+LobbyAction_crash#0,"""Ｔ\u3000ｅ\u3000ｒ\u3000ｔ\u3000ａ\u3000ｂ\u3000ｒ\u3000ａ\u3000ｋ"""
```
can you remove the fullwidth space (\u3000) and convert the `Tertabrak`

alama reviewed (changes requested):
fullwidth space are a mess

DOTzX committed:
ID: Up

DOTzX committed:
ID: Up 2

DOTzX committed:
ID: Up 3

DOTzX committed:
ID: Generalizing

Well, i'm new about how "this" work XD
Revert to basic/default for generalizing...

helenoa committed:
ID

Stuffs translated by me - Helenoa

helenoa committed:
Some translated files deleted from Files folder.

helenoa committed:
ID: Update 11/30 3JST

Some small translation updates. Some might have translations wrong though.

DOTzX committed:
Update wh_visifon.csv

Replace EN string to JP

DOTzX committed:
Update do00001009.csv

DOTzX committed:
Update do00001009.csv

DOTzX committed:
Update do00001000.csv

DOTzX committed:
Update co_013318.csv

Replace EN string to JP

DOTzX committed:
Update cm_013508.csv

Replace EN String to JP

DOTzX committed:
Update cm_013509.csv

Replace EN String to JP

DOTzX committed:
Update bn_000002.csv

Replace EN String to JP

DOTzX committed:
Update cm_013501.csv

Replace EN String to JP

DOTzX committed:
Update cm_013503.csv

Replace EN String to JP

DOTzX committed:
Update cm_013504.csv

Replace EN String to JP

DOTzX committed:
Update cm_013505.csv

Replace EN String to JP

DOTzX committed:
Update cm_013506.csv

Replace EN String to JP

DOTzX committed:
Update cm_013507.csv

Replace EN String to JP

DOTzX committed:
Merge pull request #3 from helenoa/ID

Update 11/29.
Don't use EN linestring, when EN file already. Use JP.
Everything is OK :ok_hand:

DOTzX committed:
ID: Update

Fix bad quote.
Note to self: ui_guild 200 line translated

DOTzX committed:
ID: Update 2

Fix quote issue

DOTzX committed:
ID: Update 3

Replace EN string to JP

alama committed:
ID: resort files
